src/components/editor/PropertyLabel.jsx

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import shortid from 'shortid'
+import React, {Component} from 'react'
+import { OverlayTrigger, Popover } from 'react-bootstrap'
+import PropTypes from 'prop-types'
+import RequiredSuperscript from './RequiredSuperscript'
+
+export class PropertyLabel extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render () {
+
+    let title = []
+    const key = shortid.generate()
+    const property = this.props.pt
+
+    const popover = (
+      <Popover id="popover-basic" title={property.label} >
+        {property.remark}
+      </Popover>
+    )
+
+    try {
+      const url = new URL(property.remark)
+      title.push(
+        <a href={url} className="prop-remark" alt={property.remark} key={key} >
+          <span className="prop-remark">{property.propertyLabel}</span>
+        </a>
+      )
+    } catch (_) {
```

Comment:
is this catch a lodash thing (underscore)?  no lodash import ...

Reply:
I'm a bit :unamused: about bare `catch`es, but I acknowledge there is a place and time for everything. Can you say a bit about what the error conditions are here?

Reply:
I didn't write this....this was @jermnelson 's baby, but it looks like the error would be something like a failure to instantiate a new URL object. I could probably use some kind of turnery to handle it if you want to see that attempt?

Reply:
@jgreben 	that would be lovely!

Reply:
I changed it to be a kind if turnery. LMK what you think.